src/zos/app/flag.rs

```diff
@@ -0,0 +1,10 @@
+use std::path::Path;
+
+pub const FLAGS_DIR: &str = "/tmp/flags";
+// LimitedCache represent the flag cache couldn't mount on ssd or hdd
+pub const LIMITED_CACHE: &str = "limited-cache";
+
+// CheckFlag checks the status of a flag based on a key
+pub fn check_flag(key: String) -> bool {
```

Comment:
why does this take a String? also the name is wrong because this is already a `flag` sub module so when u call it it will be `flag::check_flag` which too long instead this can be 

```suggestion
pub fn check<S: AsRef<str>>(key: S) -> bool
```

another function `set` and `unset` need to be added